services/wordpress/wordpress-downloads.service.js

```diff
@@ -89,16 +89,14 @@ function InstallsForExtensionType(extensionType) {
     static get route() {
       return {
         base: `wordpress/${extensionType}/installs`,
-        format: '(.+)',
-        capture: ['slug'],
+        pattern: ':slug',
```

Comment:
Will using the default pattern cause any issues with slugs that contain a `/`? I'm not sure what slugs look like/are allowed to contain but just wanted to double check given the previous expression

Reply:
I looked on the wordpress site and it doesn't look like they have slashes in them. It's possible digging further could have revealed some. If we go too restrictive, we'll find out and can add a test.

It occurred to me that _git branches can have slashes in them_. There are probably a lot of those which need updating in here…

Reply:
Oh wow. Good catch, and yeah I think there will be a _lot_ of those patterns.

Reply:
It looks like you got all of these already right?

Reply:
Yes, I think so.